### PR TITLE
Add real time notification test app 

### PR DESCRIPTION
**Pull request by marche0133**
To follow the iOS sdk, we put this test app into SDK repo. Originally this app is for internal testing of real time notification (signaling) purpose. We may update this app furthermore to make it into a demo app that demo chat SDK functionalities in future.

### CODE REVIEW

**Review by vcolin7 — APPROVED** [review]
In general this looks pretty good to me, I haven't had the time to download and test it but I'm approving it to unblock for the time being. Left a couple questions and suggestions I would like you to consider before merging :)

File: samples/sample-chat-app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    id 'com.android.application'
+}
+
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        applicationId "com.azure.android.communication.sample_chat_app"
```
[the remark this answers is not in the document]
Comment (vcolin7) [reply samples/sample-chat-app/build.gradle:11]: I don't think we currently have guidance for adding samples in a specific package, these are mere suggestions from the top of my head.

File: samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java [hunk 2]
```diff
@@ -17,19 +32,27 @@
...
 public class MainActivity extends AppCompatActivity {
 
     private ChatClient chatClient;
     private int eventHandlerCalled;
     private JSONObject eventPayload;
 
     // Replace <userToken> with your valid communication service token
-    private final String userAccessToken = "<userToken>";
-    private final String endpoint = "https://chat-sdktester-e2e.int.communication.azure.net";
+    private final String userAccessToken = "eyJhbGciOiJSUzI1NiIsImtpZCI6IjEwMl9pbnQiLCJ4NXQiOiJnMTROVjRoSzJKUklPYk15YUUyOUxFU1FKRk0iLCJ0eXAiOiJKV1QifQ.eyJza3lwZWlkIjoiYWNzOjQ2ODQ5NTM0LWViMDgtNGFiNy1iZGU3LWMzNjkyOGNkMTU0N18wMDAwMDAwOC03YjVhLThiNzgtMTY1NS0zNzNhMGQwMDliYTEiLCJzY3AiOjE3OTIsImNzaSI6IjE2MTQyOTQ5MzYiLCJpYXQiOjE2MTQyOTQ5MzYsImV4cCI6MTYxNDM4MTMzNiwiYWNzU2NvcGUiOiJjaGF0IiwicmVzb3VyY2VJZCI6IjQ2ODQ5NTM0LWViMDgtNGFiNy1iZGU3LWMzNjkyOGNkMTU0NyJ9.e03hLzcAoXWYW_lyMmK8ixf5s4K7k3gIccOs7Ykm5JUvA57CgrZx8SD19uM4NI2dq8O2Yhuf_p1RX7UyT1VJmuS6_zqLKtJfq2eqNTErFcb9d2UjjL8rJPr_TJMX3CZBxi65FgSrIUu1dHGFjkR4EVsViGo64zbKl3XDfMdFlgessg6jAfHCsBsD6XUMhl54w3GYmLtKydpyAaHzaqRBjSpLaZZeFucPAvs86COX_bqhYzg93bLel5iDKWmAI1bl0sFFqcDdPYwpiypWn2xKq5XXwHc9a7gpwiAldEAzQDC7O6e32AtoDfYWppaB5bTDg4PyiP35y_yiM3WH-y_QvQ";
```
Comment (vcolin7) [inline samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java:47]: Is this a valid token? We do not want to commit credentials to our repositories at any point in time.

Comment (marche0133) [reply samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java:47]: Token expires in 24 hours, but this one I accidentally commit, let me revert it.

**Pull request merged**